debian/control

```diff
@@ -31,5 +33,4 @@ Depends: ${misc:Depends},
          python3-retrying,
 Description: Test release upgrades in a virtual environment
  A tool to do QA for release upgrades in ubuntu that performs upgrades
```

Comment:
I think the description paragraph would read better as "A tool for performing release upgrades in a virtual environment". The existing one seems excessively wordy.